assets/js/script.js

```diff
@@ -164,6 +164,11 @@ $(document).ready(() => {
     if (_.includes(keys, 'KEYMAP')) {
       return 'KEYMAP';
     }
+    // avoid keymaps ending with _kc unless we have no other choice
```

Comment:
IIRC, these types of keymaps don't work at all in the Configurator. I think we should exclude them completely unless we're planning to support them in future, which I don't think we are.

Also, I know a couple keyboards have these types of keymaps, but the macro *contains* `_kc_` rather than ending in `_kc`. I know the Contra is one of those.